### DISCUSSION TITLE
Add statement of cash flows

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5833] [Design]
Moved from https://sourceforge.net/p/ledger-smb/feature-requests/199/

With the new account hierarchy reporting system we should be able to select hierarchies for the sections of the reports properly.  For 1.4 and 1.5 this could be report-time selection.  At some point we could allow system configuration for hierarchies and this report.

**Comment by ehuelsmann** [comment]
Would be nice to set up a list of requirements to fulfill https://corporatefinanceinstitute.com/resources/knowledge/accounting/statement-of-cash-flows/
